### PR TITLE
Patron aaa aplicado

### PR DESCRIPTION
**Pull request by JuanManu59**

### CODE REVIEW

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-const shopingCartPage = new ShopingCartPage();
-const loginPage = new LoginPage();
-const addressStepPage = new AddressStepPage();
-const shippingStepPage = new ShippingStepPage();
-const paymentStepPage = new PaymentStepPage();
-
-const email = "aperdomobo@gmail.com";
-const password = "WorkshopProtractor";
+import {MenuContentPage, ProductsListPage, ShopingCartPage,
+  LoginPage, AddressStepPage, ShippingStepPage, PaymentStepPage} from "../page/index";
```
Comment (danielgalvis98) [inline cypress/e2e/buy-shirt.cy.ts]:
```suggestion
import {
  MenuContentPage,
  ProductsListPage,
  ShopingCartPage,
  LoginPage,
  AddressStepPage,
  ShippingStepPage,
  PaymentStepPage} from "../page/index";
```

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
 
 describe("Buy a t-shirt", () => {
+  let menuContentPage:MenuContentPage;
+  let productsListPage:ProductsListPage;
+  let shopingCartPage:ShopingCartPage;
+  let loginPage:LoginPage;
+  let addressStepPage:AddressStepPage;
+  let shippingStepPage:ShippingStepPage;
+  let paymentStepPage:PaymentStepPage;
```
Comment (danielgalvis98) [inline cypress/e2e/buy-shirt.cy.ts]:
```suggestion
  let menuContentPage: MenuContentPage;
  let productsListPage: ProductsListPage;
  let shopingCartPage: ShopingCartPage;
  let loginPage: LoginPage;
  let addressStepPage: AddressStepPage;
  let shippingStepPage: ShippingStepPage;
  let paymentStepPage: PaymentStepPage;
```

File: cypress/page/dresses-list.page.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  
+    validateItemsNames(names: string[]){
+      cy.get(this.dressName).each((item, index) => {
+        cy.wrap(item).should("contain.text", names[index])
+      })
+    }
+  
+  }
+  
+  export {DressesListPage}
```
Comment (danielgalvis98) [inline cypress/page/dresses-list.page.ts]:
```suggestion
export {DressesListPage}
```
and EOF.

File: cypress/page/index.ts [hunk 4]
```diff
@@ -5,3 +5,4 @@ export {LoginPage} from "./login.page";
 export {AddressStepPage} from "./address-step.page";
 export {ShippingStepPage} from "./shipping-step.page";
 export {PaymentStepPage} from "./payment-step.page";
+export {DressesListPage} from "./dresses-list.page";
```
Comment (danielgalvis98) [inline cypress/page/index.ts]: EOF

File: cypress/page/dresses-list.page.ts (hunk 3 above)
Comment (danielgalvis98) [reply cypress/page/dresses-list.page.ts]: In general check the identation of this class, seems like a mix of 4 and 2 spaces.

File: cypress/e2e/buy-shirt.cy.ts (hunk 1 above)
Comment (JuanManu59) [reply cypress/e2e/buy-shirt.cy.ts]: resolved

File: cypress/e2e/buy-shirt.cy.ts (hunk 2 above)
Comment (JuanManu59) [reply cypress/e2e/buy-shirt.cy.ts]: resolved

File: cypress/page/dresses-list.page.ts (hunk 3 above)
Comment (JuanManu59) [reply cypress/page/dresses-list.page.ts]: resolved

File: cypress/page/index.ts (hunk 4 above)
Comment (JuanManu59) [reply cypress/page/index.ts]: resolved

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**